src/flowMC/sampler/Sampler.py

```diff
@@ -495,3 +454,48 @@ def save_summary(self, path: str):
         """
         with open(path, "wb") as f:
             pickle.dump(self.summary, f)
+
```

Comment:
It was an explicit design choice to not include any plotting in the main sampler. We can look into adding a module for analysis and postprocessing in utils.

Reply:
I have moved these to a postprocessing file in utils.